vendor/github.com/integr8ly/cloud-resource-operator/pkg/providers/aws/credentials.go

```diff
@@ -24,7 +24,7 @@ const (
 	defaultProviderCredentialName = "cloud-resources-aws-credentials"
 
 	defaultCredentialsKeyIDName = "aws_access_key_id"
-	// #nosec G101
+	// #nosec G101 -- This is a false positive
```

Comment:
Running `make vendor/fix` seems to prefer without the comment, which I think would make sense since this is in the `vendor` package, so we're just pulling in the current CRO version dependency files :thinking:

Reply:
@KevFan good spot thanks for the tip, test's passing now :slightly_smiling_face:

Reply:
Yup, I think we can add the comment to CRO here instead if we want to add it in https://github.com/integr8ly/cloud-resource-operator/blob/master/pkg/providers/aws/credentials.go#L27 :slightly_smiling_face: